packages/smooth_app/lib/pages/page_manager.dart

```diff
@@ -98,6 +99,14 @@ class PageManagerState extends State<PageManager> {
 
   Widget _buildOffstageNavigator(BottomNavigationTab tabItem) {
     final bool offstage = _currentPage != tabItem;
+    //If is ScanTab check if it is currently offstage or not to decide whether the camera should be active or not
+    if (tabItem == BottomNavigationTab.Scan) {
+      if (offstage) {
+        ScanPageState.continuousScanModel?.pauseQRView();
+      } else {
+        ScanPageState.continuousScanModel?.restartQRView();
```

Comment:
I don't like this for because the `PageManager` shouldn't give a damn about what the scan page should actually do in a pause or in a resume state, therefore not know about `continuousScanModel`, `pauseQRView` and `restartQRView`.

Beyond the `static` problem (see my other comment), I would rather play it like this: `ScanPageState.onPause()` and `ScanPageState.onResume()`.